test/e2e/top_test.go

```diff
@@ -89,6 +90,12 @@ var _ = Describe("Podman top", func() {
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
 		Expect(len(result.OutputToStringArray())).To(BeNumerically(">", 1))
+
+		result = podmanTest.Podman([]string{"top", session.OutputToString(), "uid", "huid"})
+		result.WaitWithDefaultTimeout()
+		Expect(result).Should(Exit(0))
+		Expect(len(result.OutputToStringArray())).To(BeNumerically(">", 1))
```

Comment:
Can we run two times top on the container, once with `uid` another time with `huid`?  I really want to have a look at the returned values. `uid` should return 0.  `huid` should return `id -u`.

Reply:
Fixed.